projects/docs/docs/manifest/api-reference.md

```diff
@@ -0,0 +1,12 @@
+---
+title: ProjectDescription.framework
+slug: '/manifest/api-reference'
+description: 'This page collects manifest API references.'
+---
+
+The API documentation for the latest version and main branch is available here:
+
+* [latest](https://tuist.github.io/tuist/latest/documentation/projectdescription/)
+* [main](https://tuist.github.io/tuist/main/documentation/projectdescription/)
+
+Other versions can be found at https://tuist.github.io/tuist/VERSION/documentation/projectdescription/ by replacing `VERSION` with a version number (like X.Y.Z)
```

Comment:
does docc have an option to switch between versions as docusaurus does instead of having to manually change the URL?

Reply:
Not currently. Initially I was trying to generate a list of all the version but I couldn't do it.